src/compiler/moduleNameResolver.ts

```diff
@@ -1770,6 +1770,12 @@ function nodeModuleNameResolverWorker(features: NodeResolutionFeatures, moduleNa
                 resolved = loadModuleFromSelfNameReference(extensions, moduleName, containingDirectory, state, cache, redirectedReference);
             }
             if (!resolved) {
+                if (moduleName.indexOf(":") > -1) {

```

Comment:
I think this condition needs to be more specific.. 
theorotically module name could be `"c:./filename"` ?

Reply:
Sorry i keep forgetting the rooted disk path are "relative module name"